src/components/move/MoveMenuItem.tsx

```diff
@@ -18,22 +16,25 @@ export const FileMenuItem = ({
   isResource = false,
 }: FileMenuItemProps): JSX.Element => {
   return (
-    <div
-      id={id}
+    <Box
+      w="100%"
       data-cy={id}
-      className={`
-        ${elementStyles.dropdownItemDisabled}
-      `}
+      color="grey.200"
+      pl="1rem"
+      pr="1.25rem"
+      py="0.75rem"
     >
-      <Icon w="24px" h="24px" as={BxFileBlank} mr="1.5rem" />
-      {pageFileNameToTitle(name, isResource)}
-    </div>
+      <HStack spacing="1rem">
+        <Icon as={BiFileBlank} fontSize="1.25rem" />
+        <Text textStyle="body-1">{pageFileNameToTitle(name, isResource)}</Text>
+      </HStack>
+    </Box>
   )
 }
 
 export interface DirMenuItemProps {
   name: string
-  id: string
+  id: string | number
```

Comment:
hmmmm  wait why got 2 data types ah

Reply:
Hmm cos the ID can come from the index when doing a `.map()`, which is a `number`.